composer.json

```
@@ -15,7 +15,7 @@
         "ext-intl": "*",
         "ext-json": "*",
         "ext-openssl": "*",
-        "allure-framework/allure-codeception": "~1.4.0",
+        "allure-framework/allure-codeception": "~1.5.0",
```

Also for backward compatibility we might keep something like this:
```suggestion
        "allure-framework/allure-codeception": "~1.4.0||~1.5.0",
```